api/turing/config/config.go

```diff
@@ -354,6 +356,21 @@ func (c *OpenapiConfig) SpecData() ([]byte, error) {
 	return utils.MergeTwoYamls(c.SpecFile, c.SpecOverrideFile)
 }
 
+// GenerateSpecFile gets the spec data and writes to a file
+func (c *OpenapiConfig) GenerateSpecFile() error {
+	b, err := c.SpecData()
+	if err != nil {
+		return err
+	}
+
+	err = os.MkdirAll(filepath.Dir(c.MergedSpecFile), 0755)
```

Comment:
minor, but wouldn't `0644` be enough here?

Reply:
I think it would be enough, but the rest of the folders in the image are 755 so I thought I would keep it consistent.